package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@yext/search-ui-react",
-  "version": "1.5.0",
+  "version": "1.6.0-beta.432",
```

Comment:
can you also update the test-site/package-lock?

Reply:
sure!